metagraph_numba/compiler.py

```diff
@@ -1 +1,320 @@
+"""
+Numba compiler plugin:
+
+This compiler plugin assumes that concrete algorithm implementations are
+"Numba-ready".  Specifically:
+
+* The decorated Python function is compilable by Numba.
+* The input and output data types can be boxed/unboxed by Numba automatically.
+
+To created a fused multiple tasks into a subgraph, all of the individual task
```

Comment:
I think we can rephrase "To created a fused multiple tasks into a subgraph" to be more concise.